libs/remix-simulator/src/vm-context.ts

```diff
@@ -124,6 +88,158 @@ class StateManagerCommonStorageDump extends DefaultStateManager {
   }
 }
 
+export interface CustomEthersStateManagerOpts {
+  provider: string | ethers.providers.StaticJsonRpcProvider | ethers.providers.JsonRpcProvider
+  blockTag: bigint | 'earliest',
+  /**
+   * A {@link Trie} instance
+   */
+  trie?: Trie
+}
+
+class CustomEthersStateManager extends StateManagerCommonStorageDump {
+  private provider: ethers.providers.StaticJsonRpcProvider | ethers.providers.JsonRpcProvider
+  private blockTag: string
```

Comment:
`blockTag` type definition (`string`) does not match it's type definition on line 93 (`bigint | 'earliest'`)